ansible/roles-infra/infra-osp-template-create/.yamllint

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Why are we adding custom .yamllints everywhere? Isn't the one at the AgnosticD root level enough?

Reply:
We don't yet have the codebase to the level where it could pass a yamllint at the root level. We don't even have it to a place where it could pass for `ansible/roles-infra/`. So we continue to go bit by bit, working towards being able to run lint at a higher level.